test/test_macho.rb

```diff
@@ -5,6 +5,12 @@
 class MachOFileTest < Minitest::Test
   include Helpers
 
+  def test_empty_file
+    assert_raises MachO::NotAMachOError do
+      MachO::MachOFile.new_from_bin("\x00")
```

Comment:
I think this should be just `""` (an empty string) to stay true to the name of the test.